Makefile

```diff
@@ -0,0 +1,80 @@
+deps: composer.json
```

Comment:
Did you mean composer.lock?

Then it must be like:
```
deps: composer.lock
composer.lock: composer.json
    composer update
```
Else it will run `composer install` on every build.